core/src/main/java/tachyon/client/TachyonFile.java

```diff
@@ -443,14 +446,15 @@ private TachyonByteBuffer readLocalByteBuffer(int blockIndex, long offset, long
 
   /**
    * Get the the whole block from remote workers.
-   * 
+   *
    * @param blockInfo The blockInfo of the block to read.
    * @return TachyonByteBuffer containing the block.
    */
   TachyonByteBuffer readRemoteByteBuffer(ClientBlockInfo blockInfo) {
     // We call into the remote block in stream class to read a remote byte buffer
-    ByteBuffer buf = RemoteBlockInStream.readRemoteByteBuffer(mTachyonFS,
-        blockInfo, 0, blockInfo.length, mTachyonConf);
+    ByteBuffer buf =
```

Comment:
This was done by my eclipse formatter using the formatter file provided on tachyon webpage.